scripts/seed-db.sh

```diff
@@ -0,0 +1,14 @@
+set -e
```

Comment:
this doesn't run on my machine without a shebang on the first line:
```sh
#!/bin/sh
```

Reply:
I'm pretty sure my environment attempts to run zsh without posix compatibility mode enabled without this shebang

Reply:
yeah, makes sense we'd need to specify sh or bash here. i always have to do this because i run [fish](https://fishshell.com/) on my machine